front-end/src/mocked-members.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "name": "Member One",
+    "email": "member.one@example.com",
+    "team": "ReactDojo",
```

Comment:
During our meeting, it was noted that the "team" field would reference a Mongo ID from the teams collection.

Reply:
If we are going to just throw the changes into our app for now, we could add some fake ids.